MC-40589: Document page builder preview limitation for filtering/sorting configurable, grouped and bundle products by price. (#1016)
magento/merchdocs: src/cms/page-builder-add-content-products.md

   {:.bs-callout-info}
   Some products in the Category, SKU, or Condition settings may not appear in your products grid or carousel. For example, disabled products, products marked as not visible, out-of-stock products, and products assigned to another website will not be displayed.

   {:.bs-callout-warning}

>    Configurable, grouped and bundle (dynamic price) products price is undefined in admin area. Therefore, such products will not be displayed in the **preview** if the products are filtered by price. Also, such products will not be ordered correctly in the **preview**, if the products are ordered by price.

Reviewer: ```suggestion
   Prices for configurable, grouped and bundled (dynamic price) products are undefined in the Admin area. Therefore, these products are not displayed in the **preview** if the products are filtered by price. These products cannot be ordered correctly in the **preview** if ordered by price.
```
